rules/macos/credential_access_mitm_localhost_webproxy.toml

```diff
@@ -34,7 +34,10 @@ event.category:process and host.os.type:macos and event.type:start and
  process.name : networksetup and process.args : (("-setwebproxy" or "-setsecurewebproxy" or "-setautoproxyurl") and not (Bluetooth or off)) and
  not process.parent.executable : ("/Library/PrivilegedHelperTools/com.80pct.FreedomHelper" or
                                   "/Applications/Fiddler Everywhere.app/Contents/Resources/app/out/WebServer/Fiddler.WebUi" or
-                                  "/usr/libexec/xpcproxy")
+                                  "/usr/libexec/xpcproxy") and
+ not process.Ext.effective_parent.executable : ("/Applications/GreenVPN.app/Contents/MacOS/GreenVPN", 
+                                                "/Applications/Proxyman.app/Contents/MacOS/Proxyman",
+                                                "/private/var/folders/*/AppTranslocation/*/Fiddler Everywhere.app/Contents/MacOS/Fiddler Everywhere")
```

Comment:
This one is not going to work, as its a KQL query, wildcards will not work when put between quotes.